client/src/cb-admin/pages/Settings.js

```diff
@@ -112,15 +123,69 @@ export default class SettingsPage extends React.Component {
           this.props.history.push('/admin/login');
         }
       });
+
+    Visitors.get(this.props.auth)
```

Comment:
Why grab all this data on page load? This data is only required in the user clicks the export button. It increases the load on the server/database unnecessarily.